README.md

```diff
@@ -2,95 +2,75 @@
 
 UMCCR Orchestration Bus that leverage AWS EventBridge as Event Bus to automate the BioInformatics Workflows Pipeline.
 
-## Quickstart
+## Development
 
-### Pre-requisites
+### Toolchain
 
 ```
-brew tap aws/tap
-brew install aws-sam-cli
+docker --version
+>Docker version 23.0.5
 
-sam --version
-SAM CLI, version 1.43.0
-
-npm install -g aws-cdk
-
-cdk version
-2.19.0 (build e0d3e62)
-```
-
-### Pre-commit hooks
-
-We are using pre-commit hooks to check for any secret-key leaks and for linting purposes.
-Check [UMCCR wiki](https://github.com/umccr/wiki/blob/master/computing/dev-environment/git-hooks.md) for details.
-
-```
-brew install pre-commit detect-secrets
+conda create -n orcabus python=3.10
```

Comment:
what's condas role is all this?

Reply:
Conda is mainly for local dev Python environment. It could be `python3 -m venv venv` alternatively; sure.

Reply:
Doesn't it need to match whatever package manager is being used for the individual python lambdas? If people are installing things into a generic conda env their actual deployed code may not have the same packages?

Reply:
Yes, it needs to match. It is more like; a high level contract/assumption such that in the current product iteration life cycle, the major version must any of 3.10 tree.

Package dependencies versions are pinned. They must work in some generic Python 3.10 environment as well as in the target Lambda runtime; which should be 3.10 as well. While having mindful with any native binding Python package in use.

CodePipeline unit/integration test cases to reinforce this.